app/imports/api/smartContract/smartContractDeployment.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash');
+
+export function bothSigned(contract) {
+  const hSignature = contract.homeownerSignature;
+  const tSignature = contract.tenantSignature;
+  const hNotSigned = hSignature === '';
+  const tNotSigned = tSignature === '';
+  console.log(`h: not signed: ${hNotSigned} signature: ${hSignature}`);
+  console.log(`t: not signed: ${hNotSigned} signature: ${hSignature}`);
+  if (hNotSigned || tNotSigned) {
+    return false;
+  }
+  return true;
```

Comment:
Negated logic always seems more difficult to me, lol . I think working with `signed = signature !== ''` and returning `hSigned && tSigned` would be easier to understand, but it's up to you. Both works. 

```
return hSigned && tSigned